cluster/gke/config-test.sh

```diff
@@ -20,6 +20,7 @@ NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
 IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-container_vm}"
 ENABLE_KUBERNETES_ALPHA="${KUBE_GKE_ENABLE_KUBERNETES_ALPHA:-}"
+NODE_OS_DISTRIBUTION=gci
```

Comment:
That's kind of interesting because all the test-infra gke gci jobs pass in "gci"